api/v1alpha1/ragengine_types.go

```diff
@@ -62,6 +62,18 @@ type InferenceServiceSpec struct {
 	// AccessSecret is the name of the secret that contains the service access token.
 	// +optional
 	AccessSecret string `json:"accessSecret,omitempty"`
+	// ContextWindowSize defines the combined maximum of input and output tokens that can be handled by the LLM in a single request.
```

Comment:
Is this optional? or required.

Reply:
It is required, the user needs to supply this so we know the valid context window size